taskDetails.php

```diff
@@ -50,46 +55,79 @@
 
         <div class="col">
             <label for="taskTypeWork"> Type of Work</label>
-           <input style="height:50px;width:160px;font-size:14pt;"type="number" class="form-control" name="taskTypeWork">
+           <input style="height:50px;width:160px;font-size:14pt;"type="text" class="form-control" name="taskTypeWork">
            <small id="" class="form-text text-muted">Detail of work type</small>
         </div>
         <div class="col">
             <label for="taskDateDue"> Due Date</label>
-            <input style="height:50px;width:150px;font-size:14pt;"type="number" class="form-control" name="taskDateDue">
+            <input class="form-control" type="datetime-local" value="2021-08-19T13:45:00" name="taskDueDate" id="taskDueDate">
             <small id="" class="form-text text-muted">Due date of task</small>
         </div>
     </div>
     </div>
         <div class="form-group form-check">
 
         </div>
-        <button type="submit" class="btn btn-primary" name="submit">Submit</button>
+        <button type="submit" class="btn btn-primary" name="submit" value="Submit">Submit</button>
     </form>
 
 
 
 </head>
 <?php
 //This file contains the php code for inserting tast detail data from the html form
-
-//declaring local php variables 
+if (!empty($_POST['submit'])){
+    //declaring local php variables 
 $submitted_taskName = $_POST['taskTitle'];
 $submitted_taskDetail = $_POST['tastDecrip'];
 $submitted_taskEffort = $_POST['taskEffort'];
 $submitted_taskDateDue= $_POST['taskDateDue'];
 $submitted_taskTypeWork= $_POST['taskTypeWork'];
 $submitted_taskListAssigned = $_POST['listID'];
+$submitted_courseID = $_POST['courseID'];
```

Comment:
Perhaps we can simplify the variable names later? You just re-declared the variable names here (some typos too), why not just use the $_POST variables seperately?